scripts/certmaker.sh

```diff
@@ -1,42 +1,35 @@
-# Generate CA private key
-openssl genpkey -algorithm RSA -out ca-key.pem
-
-# Generate CA self-signed certificate
-openssl req -new -x509 -key ca-key.pem -out ca-cert.pem -subj "/C=GB/ST=London/O=Perseus CA/CN=perseus-demo-fapi.ib1.org"
+# Perseus Trust Framework CA 
```

Comment:
certmaker.sh now aligns more closely to the proposed certificates that will be issued by the Perseus directory.